utils/experiment_options/experimentoptions.py

```diff
@@ -100,59 +96,42 @@ def __init__(self):
-        # one that expires first.
-        if len(lines) > 1:
-            times = [[line.split()[1], line.split()[2]] for line in lines]
-            min_year = times[0][0]
-            min_yrsec = times[0][1]
-            hdw_index = 0
-            for i in range(len(times)):
-                year = times[i][0]
-                yrsec = times[i][1]
-                if year < min_year:
-                    hdw_index = i
-                elif year == min_year:
-                    if yrsec < min_yrsec:
-                        hdw_index = i
-            hdw = lines[hdw_index]
-        else:
-            try:
-                hdw = lines[0]
-            except IndexError:
-                errmsg = 'Cannot find any valid lines for this time period in the hardware file ' \
-                         '{}'.format((hdw_dat_file + self.site_id))
-                raise ExperimentException(errmsg)
+
+        # Take the final line
+        try:
+            hdw = lines[-1]
+        except IndexError:
+            errmsg = 'Cannot find any valid lines in the hardware file: ' \
+                     '{}'.format((hdw_dat_file + self.site_id))
+            raise ExperimentException(errmsg)
         # we now have the correct line of data.
 
         params = hdw.split()
-        if len(params) != 19:
-            errmsg = 'Found {} parameters in hardware file, expected 19'.format(len(params))
+        if len(params) != 22:
+            errmsg = 'Found {} parameters in hardware file, expected 22'.format(len(params))
             raise ExperimentException(errmsg)
 
-        self._geo_lat = params[3]  # decimal degrees, S = negative
-        self._geo_long = params[4]  # decimal degrees, W = negative
-        self._altitude = params[5]  # metres
-        self._boresight = params[6]  # degrees from geographic north, CCW = negative.
-        self._beam_sep = params[7]  # degrees TODO is this necessary, or is this a min. - for
-        # post-processing software in RST? check with others.
-        self._velocity_sign = params[8]  # +1.0 or -1.0
-        self._analog_rx_attenuator = params[9]  # dB
-        self._tdiff = params[10] # ns
-        self._phase_sign = params[11]
-        self._intf_offset = [float(params[12]), float(params[13]), float(params[14])]
+        self._status = params[1]  # 1 operational, -1 offline
+        self._geo_lat = params[4]  # decimal degrees, S = negative
+        self._geo_long = params[5]  # decimal degrees, W = negative
+        self._altitude = params[6]  # metres
+        self._boresight = params[7]  # degrees from geographic north, CCW = negative.
+        self._boresight_shift = params[8]  # degrees from physical boresight. nominal 0.0 degrees
+        self._beam_sep = params[9]  # degrees, nominal 3.24 degrees
+        self._velocity_sign = params[10]  # +1.0 or -1.0
+        self._phase_sign = params[11]  # +1 indicates correct interferometry phase, -1 indicates 180
+        self._tdiff_a = params[12]  # ns for channel A.
```

Comment:
I think these tdiff values are in microseconds: https://github.com/SuperDARN/hdw/blob/main/README.txt
a few places to change in these two files

Reply:
Ah okay, we definitely got it wrong in the [previous PR](https://github.com/SuperDARNCanada/borealis/pull/295)

Reply:
Yeah I was thinking that it was probably from me haha.